content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/magnetic_dipole_time/analytic_solution.rst

```diff
@@ -55,20 +26,29 @@ For most geophysical problems however, we are interested in the step-off or tran
 The step-off response for a causal system may be given by:
 
 .. math::
-	g_-(t) = \int_{-\infty}^\infty f(\tau) \big [ 1 - u(t - \tau) \big ] d\tau = \int_0^\infty f(\tau) d\tau - \int_0^t f(\tau) d\tau = g_+ (\infty) - g_+(t) \; \; \; \textrm{for} \; \; \; t\geq 0
+	\begin{split}
+	g_-(t) = \int_{-\infty}^\infty f(\tau) \big [ 1 - &u(t - \tau) \big ] d\tau = \; ... \\
+	& \int_0^\infty f(\tau) d\tau - \int_0^t f(\tau) d\tau = g_+ (\infty) - g_+(t) \; \; \; \textrm{for} \; \; \; t\geq 0
+	\end{split}
 	:label: causal_step_off
 
 where :math:`g_+ (\infty )` represents the step-response at :math:`t = \infty`.
 Therefore, if the step-response is known for :math:`t \geq 0`, it can be used to obtain the step-off response at :math:`t \geq 0`.
 
-From Ward and Hohmann :cite:`ward1988`, the step-response for a causal system is equivalent to the following inverse Laplace transform:
+From Ward and Hohmann :cite:`ward1988`, the step-response can be obtained via the following inverse Laplace transform:
 
 .. math::
 	g_+(t) = L^{-1} \Bigg [ \frac{F(s)}{s} \Bigg ]
-	:label: step_inverse_Laplace
+	:label: step_Laplace_transform
 
 where :math:`F(s)` is obtained by replacing :math:`s=i\omega` in the system's harmonic response function.
-For the electric and magnetic fields arising from a harmonic magnetic dipole, these have :ref:`already been derived<frequency_domain_magnetic_dipole_analytic_solution>`.
+For the electric field, magnetic field and vector potential arising from a harmonic magnetic dipole in the :math:`\hat x` direction, these have :ref:`already been derived<frequency_domain_magnetic_dipole_analytic_solution>`.
```

Comment:
I think notation-wise, we should be using 
```
:math:`\mathbf{\hat{x}}
```
as a unit vector is a vector. Thoughts?